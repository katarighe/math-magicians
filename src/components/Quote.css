.quote {
  display: flex;
  flex-direction: column;
  padding: 40px;
  justify-content: center;
  align-items: center;
  margin: 20px;
  font-size: 1.5rem;
}

.header_2 {
  display: flex;
  align-items: center;
  justify-content: center;
  font-size: 3rem;
  animation: header-animation 2s infinite;
  color: #15f144;
}

@keyframes header_2-animation {
  0% {
    transform: translateY(0);
  }

  50% {
    transform: translateY(-10px);
  }

  100% {
    transform: translateY(0);
  }
}

.description {
  font-family: 'Segoe UI', sans-serif;
  padding: 10px;
  font-size: 1.5rem;
  text-align: center;
}

.author {
  font-family: 'Segoe UI', sans-serif;
  font-weight: bold;
  line-height: 50%;
  font-size: 1.75rem;
}
