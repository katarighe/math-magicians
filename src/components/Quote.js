import React from 'react';
import Header from './Header';
import './Quote.css';

const Quote = () => {
  <Header />;
  return (
    <div className="quote">
      <p className="description">
        Mathematics is the language of science.
        It is the language of logic and reason.
        It is the language of the universe.
        Without mathematics, we would not be able
        to understand the world around us. We would
        not be able to make predictions about the future.
        We would not be able to create new technologies.
        Mathematics is essential to our understanding
        of the world and our ability to shape it.
      </p>
      <big>
        <b className="author">
          Paul Erd≈ës, a Hungarian mathematician known to be
          one of the greatest mathematicians of the 20th century
        </b>
      </big>
    </div>
  );
};

export default Quote;
