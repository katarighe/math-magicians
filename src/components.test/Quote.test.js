import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import Quote from "../components/Quote";

test("Snapshot test for Quote component", () => {
  const component = renderer.create(<Quote />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
