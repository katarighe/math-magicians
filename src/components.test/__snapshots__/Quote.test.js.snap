// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`If it renders correctly 1`] = `
<div
  className="calculator_header"
>
  <h1
    className="header_2"
  >
    Read this favorite math quote!
  </h1>
  <div
    className="quote"
  >
    <p
      className="description"
    >
      Mathematics is the language of science. It is the language of logic and reason. It is the language of the universe. Without mathematics, we would not be able to understand the world around us. We would not be able to make predictions about the future. We would not be able to create new technologies. Mathematics is essential to our understanding of the world and our ability to shape it.
    </p>
    <big>
      <b
        className="author"
      >
        Paul Erdős
      </b>
    </big>
  </div>
</div>
`;

exports[`Snapshot test for Quote component 1`] = `
<div
  className="calculator_header"
>
  <h1
    className="header_2"
  >
    Read this favorite math quote!
  </h1>
  <div
    className="quote"
  >
    <p
      className="description"
    >
      Mathematics is the language of science. It is the language of logic and reason. It is the language of the universe. Without mathematics, we would not be able to understand the world around us. We would not be able to make predictions about the future. We would not be able to create new technologies. Mathematics is essential to our understanding of the world and our ability to shape it.
    </p>
    <big>
      <b
        className="author"
      >
        Paul Erdős
      </b>
    </big>
  </div>
</div>
`;
